docs/bc-registries-audit-notification-workflow.md

```diff
@@ -88,4 +88,15 @@ Specifics regarding the commands referenced in this workflow can be found here:
 
 ## Audit scripts for Aries VCR/OrgBook and BC Registries Issuer
 
-This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
\ No newline at end of file
+This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
+
+## Troubleshoot Failed Credentials Error
+
+If there are companies that dont't get re-processed because the credentials were created in the credential_log table in the event processor database. Follow the steps below to troubleshoot failed credentials error:
+
+Run the `./manage -e prod runPipeline`command and verify the credentials errors. An example regarding this is shown below in the image:
+[BC-Registries-Audit-Errors](./images/bc-registries-audit-errors.png)
```

Comment:
`[BC-Registries-Audit-Errors](./images/bc-registries-audit-errors.png)`, The image isn't linked properly for it to be displayed.  Use the other image links in the document as a reference.

Reply:
Thanks Wade, I have fixed the image link now.